apps/native-component-list/package.json

```
@@ -41,6 +41,7 @@
     "@react-native-async-storage/async-storage": "2.2.0",
     "@react-native-community/datetimepicker": "8.4.4",
     "@expo/app-integrity": "~0.1.10",
+		"expo-age-range": "~0.2.0",
```

Looks like you used tabs instead of spaces here
```suggestion
    "expo-age-range": "~0.2.0",
```